op-interop-mon/flags/flags.go

```diff
@@ -26,13 +26,23 @@ var (
 		EnvVars:  prefixEnvVars("L2_RPCS"),
 		Required: true,
 	}
+
+	// Optional Flags
+	SupervisorEndpointFlag = &cli.StringFlag{
```

Comment:
Is there a world where we want to call `admin_setFailsafeEnabled` on _multiple_ supervisors? If so, we should make this a stringslice.